src/cmd/services/m3query/config/config.go

```diff
@@ -340,7 +345,9 @@ func TagOptionsFromConfig(cfg TagOptionsConfiguration) (models.TagOptions, error
 	}
 
 	if cfg.Scheme == models.TypeDefault {
```

Comment:
super nit: Would it be cleaner to just check if this was nil?

Reply:
Yeah, I think when we take it to step 2 and make this optional would definitely check for nil instead